src/scss/_sofie-rundown.scss

```diff
@@ -0,0 +1,13 @@
+.transparent-red-zebra-opacity {
+  &:before {
+    content: '';
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background: var(--transparent-and-red-stripes);
+    z-index: 10;
+    opacity: 0.8;
+  }
+}
```

Comment:
I don't know if I am a fan of extracting the style to a common class. To me it seems like a slight overkill – having the background gradient extracted is enough (for me at least).

Reply:
We should avoid having multiple ways of doing the same thing. Just because it's CSS doesn't mean that we shouldn't apply our principles to it.

Reply:
see https://github.com/tv2/ng-sofie/pull/104#discussion_r1494403248